test/k23si/3SITxnTest.cpp

```diff
@@ -1279,6 +1285,484 @@ seastar::future<> testScenario01() {
+            dto::Key {.partitionKey = "SC02_pkey11", .rangeKey = "range12"},
+            dto::Key {.partitionKey = "SC02_pkey2", .rangeKey = "range1"},
+            DataRec {.f1="SC02_f_end", .f2="SC02_f_end"}, // new value
+            DataRec {.f1="SC02_f1", .f2="SC02_f2"},       // same value as test setup
+            [this](auto& commitMTR, auto& abortMTR, auto& k11, auto& k12, auto& v1, auto& vold) { 
+                return seastar::when_all(doWrite(k11, v1, commitMTR, k11, collname, false, true, ErrorCaseOpt::NoInjection), doWrite(k12, v1, abortMTR, k12, collname, false, true, ErrorCaseOpt::NoInjection))
+                .then([&](auto&& response) mutable {
+                    auto& [resp1, resp2] = response;
+                    // move resp out of the incoming futures sice get0() returns an rvalue
+                    auto [status1, val1] = resp1.get0();
+                    auto [status2, val2] = resp2.get0();
+                    K2EXPECT(status1, Statuses::S201_Created);
+                    K2EXPECT(status2, Statuses::S201_Created);
+                })
+                .then([this, &commitMTR, &abortMTR, &k11, &k12] {
+                    return seastar::when_all(doEnd(k11, commitMTR, collname, true, {k11}, Duration(1s),ErrorCaseOpt::NoInjection),doEnd(k12, abortMTR, collname, false, {k12}, Duration(1s),ErrorCaseOpt::NoInjection))
+                    .then([&](auto&& response) mutable {
+                        auto& [resp1, resp2] = response;
+                        // move resp out of the incoming futures sice get0() returns an rvalue
+                        auto [status1, val1] = resp1.get0();
+                        auto [status2, val2] = resp2.get0();
+                        K2EXPECT(status1, Statuses::S200_OK);
+                        K2EXPECT(status2, Statuses::S200_OK);
+                    });
+                })
+                .then([this, &k11, &k12, &commitMTR, &abortMTR] {
+                    return seastar::when_all(doFinalize(k11, k11, commitMTR, collname, false, ErrorCaseOpt::NoInjection), doFinalize(k12, k12, abortMTR, collname, true, ErrorCaseOpt::NoInjection))
+                    .then([&](auto&& response) mutable {
+                        auto& [resp1, resp2] = response;
+                        // move resp out of the incoming futures sice get0() returns an rvalue
+                        auto [status1, val1] = resp1.get0();
+                        auto [status2, val2] = resp2.get0();
+                        K2INFO("SC01.case11::OP_Finalize(abort)_a_END(commit)_key. " << "status: " << status1.code << " with MESG: " << status1.message);
+                        K2INFO("SC01.case11::OP_Finalize(commit)_a_END(abort)_key. " << "status: " << status2.code << " with MESG: " << status2.message);
+                        K2EXPECT(status1, Statuses::S405_Method_Not_Allowed);
+                        K2EXPECT(status2, Statuses::S405_Method_Not_Allowed);
```

Comment:
What do we get 405 here? I would think it should succeed. Am I misunderstanding it?